coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java

```diff
@@ -296,6 +306,11 @@ public CoapServerBuilderForUdp duplicateMsgDetectionTimeInMillis(long intervalIn
             return this;
         }
 
+        public CoapServerBuilderForUdp duplicateMessageDetectorCache(PutOnlyMap<CoapRequestId, CoapPacket> cache) {
```

Comment:
The `CoapRequestId` is an internal class, is it a good idea to expose it on the public API? Or should it be moved to the `com.mbed.coap.server` package?

Reply:
Well I moved it to com.mbed.coap.server, but I'm afraid it might have to be changed later.